portfolio/src/main/webapp/hiking.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>ROB MARCUS x HIKING</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <h1>
+      <a href="index.html" style="text-decoration: none; color: black">
+        ROB MARCUS
+      </a>
+      <div> x HIKING</div>
+    </h1>
+    <div id = "content">
+    <p>Material will go here.</p>
+    </div>
+
+    <div id = "content">
+        <div id="container">
+          <p onload="otherPages(0);" id="footer"></p>
+        </div>
+        <div id="container">
+          <div class = logo>
+            <! not sure why but the LI logo had an unsightly _ wo td = none!>
```

Comment:
FYI it's because text in \<a\> elements always have an underline and you have some whitespace, which still counts as text.